### PR TITLE
Clean up E0750

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0750.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+    type Foo;
+}
+
+default impl !MyTrait for u32 {} // error!
+# fn main() {}
+```
+
+Negative impls cannot be default impls. A default impl supplies default values
+for the items within to be used by other impls, whereas a negative impl declares
+that there are no other impls. These don't make sense to combine.
```
Comment (pickfire) [inline src/librustc_error_codes/error_codes/E0750.md:18]:
```suggestion
that there are no other impls. Combining it does not make sense.
```

**Review by pickfire — CHANGES_REQUESTED** [review]
Looks good to me other than the mentioned change.

**Review by pickfire — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
@pickfire Thanks for the review! Updated.

**Pull request merged**